engine/rpcserver.go

```diff
@@ -16,7 +16,7 @@ import (
 
 	"github.com/gofrs/uuid"
 	grpcauth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
-	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
```

Comment:
Reverting ths change, running the latest version of protoc and following the buf commands will regenerate the gRPC stubs to runtime v2.

![image](https://user-images.githubusercontent.com/4685270/126921218-bc9c7515-ee7b-4eb4-b8db-27df7e0bce7a.png)